Jenkinsfile

```diff
@@ -41,8 +41,6 @@ pipeline {
           steps {
             script { img.inside {
               sh '''
-                eval $(opam config env)
-                eval $(perl -I "~/perl5/lib/perl5" -Mlocal::lib)
```

Comment:
What's the reasoning for moving these out of the Jenkinsfile? The alternative here of duplicating these across multiple makefiles doesn't seem particularly optimal.

Reply:
I've been meaning to do this for a while now; I also saw it in the evm semantics makefiles that I've been working with in the last few days and really liked it.

It's not the primary purpose of this PR though, so if you don't agree, I'll revert this change.